src/yalexs_ble/push.py

```diff
@@ -64,20 +63,7 @@
 # there is no update from the lock.
 VALID_ADV_VALUES = {0, 1}
 
-
-def cancelable_operation(func: WrapFuncType) -> WrapFuncType:
-    """Define a wrapper to make mutually exclusive operations cancelable."""
-
-    async def _async_wrap_cancelable_operation(
-        self: "PushLock", *args: Any, **kwargs: Any
-    ) -> None:
-        await self._cancel_in_progress_operation()
-        await self._cancel_in_progress_update()
-        self._operation_task = asyncio.create_task(func(self, *args, **kwargs))
-        await self._operation_task
-        self._operation_task = None
-
-    return cast(WrapFuncType, _async_wrap_cancelable_operation)
+DISCONNECT_DELAY = 12.5
```

Comment:
This is just way too long with HomeKit, a phone, and a bridge, and HA all wanting a connection for a device that only supports 2 at a time